### DISCUSSION TITLE
Need help with home lab setup

### DISCUSSION THREAD

**Discussion by RadarG1** [Security-Onion-Solutions/securityonion#1995] [General] [answered]
I have a small test network setup with VMware using two domains connected to several vyatta vm routers. I was thinking about adding multiple interfaces to SO and add them to every vswitch running in promiscuous mode is this the best method?

**Comment by weslambert** [comment, marked as answer]
You could do this, or use a separate switch to aggregate them prior if you like.

**Reply by RadarG1** [reply]
How would go about aggregating the connections?
